models/property.py

```diff
@@ -50,7 +51,8 @@ def json(self):
     def tenants(self):
         tenants = []
         for lease in self.leases:
-            tenants.append(lease.tenant.json())
+            if lease.is_active():
+                tenants.append(lease.tenant.json())
```

Comment:
There will always be way more inactive leases than active leases. We should be able to build a query that only returns active leases, and plug that in here. So we could do something like `self.leases.active:` Once you have all the active leases you could ask the database for those tenants as well instead of iterating through all active leases and asking the db for the tenant.